New quickstart - demonstrating remote task execution with streams.

andyuk1986 opened the pull request.

andyuk1986 committed:
New quickstart demonstrating remote task execution with streams.

slaskawi commented on remote-tasks-with-streams/README.md:
```diff
@@ -0,0 +1,135 @@
+       For Windows:  COPY "target\remote-tasks-with-streams-lib.jar" "%JDG_HOME%\modules\system\layers\base\org\jboss\as\quickstarts\remote-task-with-streams\main\"
+
+* Copy the `resources/module.xml` file to `$JDG_HOME/modules/system/layers/base/org/jboss/as/quickstarts/remote-task-with-streams/main/`:
+
```
An empty line to be removed here.

slaskawi reviewed (approved):
Looks very nice! I really like this QuickStart.

@mgencur @VaclavDedik Any other comments? If not, I'll integrate it.

VaclavDedik commented:
Looks good, can you just add the quickstart to the source assembly please? It won't be packaged for JDG 7.1 if it's not there:

https://github.com/jboss-developer/jboss-jdg-quickstarts/blob/jdg-7.1.x/src/main/assemblies/sources.xml

mgencur commented:
Looks good to me ;-)

andyuk1986 commented:
Thanks everyone for the review. 
@slaskawi I have removed the extra line on the line that you have mentioned and also fixed a bit the formatting, now the README.md looks good with github viewer. 

@VaclavDedik I have added the quickstart to the source assembly as you have requested.

VaclavDedik commented:
Integrated, thanks!
